src/casp/api/server.py

```diff
@@ -166,4 +166,4 @@ async def annotate(myfile: UploadFile):
 
 
 if __name__ == "__main__":
-    uvicorn.run("server:app", host=settings.server_host, port=settings.server_port)
```

Comment:
Unless you've done some specific tuning, maybe make this default dynamic following [this recommendation](https://stackoverflow.com/questions/72374634/how-many-uvicorn-workers-do-i-have-to-have-in-production) of 2 * vCPUs + 1

```
import multiprocessing

workers=multiprocessing.cpu_count()* 2 + 1
```